### PR TITLE
Disable Digest SSP as it does not increment nonce-count properly.

### PR DESCRIPTION
**Pull request by mkhon**
This implements a workaround for #870

### CODE REVIEW

File: lib/http_digest.c [hunk 1]
```diff
@@ -116,7 +116,7 @@ CURLcode Curl_output_digest(struct connectdata *conn,
   if(!passwdp)
     passwdp = "";
 
-#if defined(USE_WINDOWS_SSPI)
+#if defined(xUSE_WINDOWS_SSPI)
```
Comment (bagder) [inline lib/http_digest.c]: Why keep code that will never be compiled and used?

**Comment by mkhon** [comment]
I am still not 100% sure that SSP Digest can not do nonce-count handling (I've read disassembly of wdigest.dll as the source is not available, and haven't checked all the code paths), so I kept the code in place.

If the change (to remove SSP Digest code) is ok I can remove it completely.

**Comment by bagder** [comment]
You tell me! If this fix makes Digest work for cases where the SSPI version does not work, it certainly sounds like an improvement to me and if so, would there be any particular downside in taking this route? (Apart from growing the libcurl size a tad bit since we need to do more work ourselves.)

**Comment by mkhon** [comment]
Daniel, the patch makes Digest fully functional and allows to use libcurl built without openssl.

I can submit the other pull-request with this code completely removed.

**Comment by bagder** [comment]
Please do. I don't think we need to keep code around that won't be used, and if someone at some point want it back, it is easy to find in git.

**Comment by mkhon** [comment]
I found a way how to generate nonce-count properly using Digest SSPI (see https://github.com/curl/curl/pull/1251)

**Pull request closed without merging**